tests/unit/preprocessor/test_runner.py

```diff
@@ -4,7 +4,7 @@
 
 def test_first_argument_name():
     """Check that the input type of all preprocessor functions is valid."""
-    valid_itypes = ('file', 'files', 'cube', 'cubes', 'products')
+    valid_itypes = ('file', 'files', 'cube', 'cubes', 'cube_list', 'products')
```

Comment:
There is no need to add `'cube_list'` here, because that is the same as `'cubes'`.